src/hooks/useHideOnScroll.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+
+type Props = {
+  handleScroll: () => void;
+};
+
+export function useHideOnScroll({ handleScroll }: Props) {
```

Comment:
this should be more generic by looking at the implementation, we could also introduce a more general hook like

https://github.com/streamich/react-use/blob/master/docs/useEvent.md

and build this one on top of it

```suggestion
export function useScrollEvent({ handleScroll }: Props) {
```

Reply:
change name and implementation :wink: